document-store/src/main/java/org/hypertrace/core/documentstore/expression/operators/AggregationOperator.java

```diff
@@ -4,6 +4,7 @@ public enum AggregationOperator {
   AVG,
   COUNT,
   DISTINCT, // This operator generates an array of distinct values
```

Comment:
Can we add a comment here that this operator (1) has been deprecated and (2) will be removed, along with the (3) alternate operator being `DISTINCT_COUNT`?
CC: @kotharironak

Reply:
In fact, we can get rid of the old one in favour of the new operator since it is a library.

Reply:
Sorry late on the review, but libraries have the same deprecation concerns as services. If it's not a lot of work, we should restore the old one and deprecate. The alternative is going make sure every consumer of the lib is up to date, which is likely more work.

Reply:
To elaborate a little bit, it's not quite the same - it takes two different forms
- compile time breakages on version upgrade. You don't want that from a library, but for an internal library this is less of a concern. It's better than runtime breakages which you can often see from a break of a service contract.
- transitives and version resolution. This is the real gotcha - If two different modules are using two different versions of doc store but running in the same VM, gradle resolves to the later version. The one that compiled against an earlier version would lead to a runtime failure. That's why java lib APIs always have the same backwards compatibility requirements as services, so we can resolve multiple libs in the same classloader depending on different versions.

Reply:
Oh okay. Understood. Will restore DISTINCT.